### DISCUSSION TITLE
Adventure to Soroban

### DISCUSSION THREAD

**Discussion by LorDDark6660** [stellar/sorobanathon#121] [General]
### Hand drawn story

https://litemint.com/items/GBUMVTA3PNBDPJCB56QYJQFFJH6H6O73W4JEYER2M447ASRBRRZRKZPG/SQLiveStory0

https://litemint.com/items/GDLTJF6IC7RMROXH4HTJ4FZK37RJ4P3UQ5XQ3F6CGXKVCHXMG7KMRUFI/SQLiveStory1

https://litemint.com/items/GDE4K3NUBCTASJXTSEG5UFDYDLJ3ERGJRNKWZ5I3VYPJIR4ANOSWVKQS/SQLiveStory2

https://litemint.com/items/GCUSU3YQDT7KOMY6WMXFDCMM4R64MPU5TWVWWO4R7QKS4VT6LQANE45F/SQLiveStory3

https://litemint.com/items/GD2PHDLUHZL3FA4SWKZCH5QP3Y3547RFPTBKVTDDN7ILZJ4T3JQVSQUJ/SQLiveStory4

https://litemint.com/items/GCT4I7PLAGLPJC4SEYDCYMJDFKPKT6WFDRTLEY24UGDBR74C5N6MONZ3/SQLiveStory5

https://litemint.com/items/GDHRPRIS5TNQKJ4E3555XQOWI2B767W6MPXS6RUQASW35M2LX45OKOXH/SQLiveStory6

https://litemint.com/items/GDVAIYMAXSJ234UEKN3VT635MCBNSKFNYRRUQESJ2IAUETBBHGBPEQA3/SQLiveStory7

**Comment by rawritude** [comment]
These are great! Thanks for sharing!

**Comment by stellarjohn** [comment]
I like these a lot!

**Comment by LorDDark6660** [comment]
Thank you, I appreciate it very much 💓✨
